sdk/metric/aggregator/lastvalue/lastvalue_test.go

```diff
@@ -76,7 +76,7 @@ func TestLastValueUpdate(t *testing.T) {
 
 		var last number.Number
 		for i := 0; i < count; i++ {
-			x := profile.Random(rand.Intn(1)*2 - 1)
+			x := profile.Random(rand.Intn(2)*2 - 1)
```

Comment:
This doesn't feel right.  What's the lint error being raised here and why is this the approach taken to fix it?

Reply:
The lint error is 
```shell
aggregator/lastvalue/lastvalue_test.go:79:24: SA4030: math/rand.Intn(n) generates a random value 0 <= x < n; that is, the generated values don't include n; rand.Intn(1) therefore always returns 0 (staticcheck)
			x := profile.Random(rand.Intn(1)*2 - 1)
			                    ^
```
As my first comment says , As a freshman for metric, I'm not sure this changes is correctly.  pls make some advice directly.